pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -1289,6 +1289,11 @@ public void deleteLedgerComplete(Object ctx) {
                 });
 
                 return deleteFuture;
+                }).whenComplete((value, ex) -> {
+                    if (ex != null) {
+                        unfenceTopicToResume();
+                        log.error("[{}] Error deleting topic", topic, ex);
```

Comment:
It is better to log before calling unfenceTopicToResume

Reply:
done